docs/DeveloperGuide.md

```diff
@@ -299,73 +290,85 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
+    * 1b1. System displays error message and does not clear command line.
+    * 1b2. User enters command with correct parameters.<br>
+  Steps 1b1-1b2 are repeated until all details entered are correct.<br>
+  Use case resumes from step 2.
 
 
 **Use case: UC02 - Read all entries**
 
 **MSS**
 1. User enters command to view all entries.
-2. System displays list with all entries to the user.
+1. System displays list with all entries to the user.
 
    Use case ends.
 
 **Extension**
 * 1a. System detects error in entered command.
     * 1a1. System displays error message and does not clear command line.
-    * 1a2. User enters new command.
-* Steps 1a1-1a2 are repeated until all details entered are correct.
-* Use case resumes from step 2.
+    * 1a2. User enters new command.<br>
+  Steps 1a1-1a2 are repeated until all details entered are correct.<br>
+  Use case resumes from step 2.<br><br>
 
 * 1b. System detects the list is empty.
-    * 1b1. System shows an empty list.
-
+    * 1b1. System shows an empty list.<br> 
   Use case ends.
 
 **Use case: UC03 - Read total earnings**
 
 **MSS**
 
-1. User enters command to read total earnings and total money owed.
-2. System displays total earnings and total money owed to the user.
+1. User enters command to read total earnings and total amount owed by the students. 
+1. System displays total earnings and total amount owed to the user.
 
    Use case ends.
 
 **Extension**
 * 1a. System detects error in entered command.
     * 1a1. System displays error message and does not clear command line.
-    * 1a2. User enters new command.
-* Steps 1a1-1a2 are repeated until all details entered are correct.
-* Use case resumes from step 2.
+    * 1a2. User enters new command.<br>
+  Steps 1a1-1a2 are repeated until all details entered are correct.<br> 
+  Use case resumes from step 2.
 
 
 **Use case: UC04 - Delete a student entry**
 
 **MSS**
 
 1. User requests to <ins>list students(UC02)</ins>.
-2. User enters command to delete a specific student.
-3. System displays list with specified student deleted from the list.
+1. User enters command to delete a specific student.
+1. System displays list with specified student deleted from the list.
 
    Use case ends.
 
 **Extensions**
 
 * 2a. System detects error in format of entered command.
     * 2a1. System displays error message and does not clear command line.
-    * 2a2. User enters new command.
-* Steps 2a1-2a2 are repeated until all details entered are correct.
-* Use case resumes from step 3.
+    * 2a2. User enters command with new index.<br>
+  Steps 2a1-2a2 are repeated until index entered is correct.<br>
+  Use case resumes from step 3.<br><br>
+
+
+* 2b. System detects error in format of entered command.
+    * 2b1. System displays error message and does not clear command line.
+    * 2b2. User enters new command.<br<
+  Steps 2b1-2b2 are repeated until all details entered are correct.<br>
+  Use case resumes from step 3.
```

Comment:
The 2 extensions here seem almost the same. Can maybe join to make just 1 extension or can specify what are the errors in the 2 extensions.

Reply:
Thanks for catching this.